client/src/Movies/SavedList.js

```diff
@@ -1,19 +1,25 @@
-import React, { Component } from 'react';
+import React, { Component } from "react"
+import { Link, NavLink } from "react-router-dom"
 
 export default class SavedList extends Component {
   constructor(props) {
-    super(props);
+    super(props)
   }
 
   render() {
     return (
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
+          <span className="saved-movie" key={movie.id}>
```

Comment:
I see that you destructured ID from the api data. Great job making that easy to read.